src/theory/quantifiers/inst_match_generator.h

```diff
@@ -642,7 +639,7 @@ class InstMatchGeneratorSimple : public IMGenerator {
   /** constructors */
   InstMatchGeneratorSimple(Node q, Node pat, QuantifiersEngine* qe);
   /** destructor */
```

Comment:
Also remove comment since destructor is removed.

Reply:
done